server/routes/executeCode.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+
+const apiMethod = (req, res) => {
```

Comment:
This logic should be written in the controllers

Reply:
Thanks! I removed all the console logs, and updated the route to use passport and moved the function to a controller.